reload/ddsdk2016
# DG-270: Show spejder.dk in node teaser

#### What does this PR do?
* DG-270: We show a 'Spejder.dk' tag if it is referenced in the `field_content_target_group`. There is a possibility of referencing unlimited amount of terms in this field, so I assumed in this PR that we're only interested in showing the spejder.dk. Let me know what you think.

Also, I started checking for the `term->getName()` but later changed it to check for `term->id()` instead.

#### How should this be manually tested?
Add "Spejder.dk" to the "Målgruppe" vocab and then add it to a node, that is shown in a teaser. It should only show "Spejder.dk" and whatever is referenced in `field_content_topic` (if any)

#### Any background context you want to provide?
From https://docs.google.com/document/d/1H3_o1Pbhvo3ychyrFpXsaXlKNQxlaOXZOlrjwLvSN1A/edit#

#### What are the relevant tickets?
DG-270

#### Screenshots (if appropriate)
See the "Spejder.dk" is shown here:
### node teaser
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/68547893/160595682-2a89d62b-1c61-4681-9abc-8d12e6c43acb.png">

### also node teaser
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/68547893/160596753-159ca3cb-3a65-4957-83ed-bdfbab3b2290.png">

### Extended teaser
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/68547893/160595328-e010df9e-fd63-41c0-ac2e-559ec1b3018f.png">

### Events 
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/68547893/160596673-97522e12-c1c4-411c-937f-e4f3cc6e05df.png">
